docs/api/public_php_api.md

```diff
@@ -1075,6 +1075,48 @@ $contentUpdateStruct->setField( 'body', $newfrenchbody );
 
 Since we don't specify a locale for the last two fields, they are set for the `UpdateStruct`'s `initialLanguageCode`, fre-FR.
 
+#### delete translations
+
+##### delete translations from a Content item version
+
+To delete translations from a Content item version, use the `deleteTranslationFromDraft` method on `ContentService`.
+
+```
+public function deleteTranslationFromDraft(VersionInfo $versionInfo, $languageCode);
+```
+
+This method returns a Content draft without the specified translation.
+
+!!! note
+
+    To remove main the translation, the main language needs to be changed manually
+    using the `ContentService::updateContentMetadata` method first.
+    Otherwise the method will throw an `\eZ\Publish\API\Repository\Exceptions\BadStateException`.
+
+
+The PHP API consumer is responsible for creating a Content item version draft and publishing it after translation removal.
+
+Since the returned Content draft is to be published, both search and HTTP cache are already handled
+by `PublishVersion` slots once the call to `publishVersion()` is made.
+
+Example:
+
+```
```

Comment:
adding `php` keyword at the end of this line should colour the syntax. Not sure if this works for ReadTheDocs, though (depends on markdown renderer)

Reply:
Even better, this now ensures ReadTheDocs will recognize the language and colour appropriately.